### ISSUE TITLE
volumeicon package with libnotify support

### ISSUE THREAD

**Issue by sarether** [void-linux/void-packages#23357]
Hello.
Make volumeicon package with libnotify support.

Thanks.

**Comment by ericonr** [comment]
It should already have notification support... Are you getting notifications from other applications?

**Comment by sarether** [comment]
Yes, of course. Notifications are present in others events.
But when I installed the volumeicon, in the preferencies there was no choice of notifications. There was only GTK+ Popup Window.
I looked for the problem in myside, but then I thought that the volumeicon was built without notifications support and then I just rename /bin/volumeicon and put one from Arch Linux package and it worked.

**Comment by ericonr** [comment]
From my build: 

```
checking for NOTIFY... yes
...
-lnotify
```

So it should have been included...

Taking a look at Arch's build: https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/volumeicon, we need a patch from https://github.com/Maato/volumeicon/commit/78a9c23775ebe12d3f3be93c0798144a6e732d90.

**Comment by ericonr** [comment]
The template then needs at least:

```diff
diff --git a/srcpkgs/volumeicon/template b/srcpkgs/volumeicon/template
index 95a0e78794..6623a09228 100644
--- a/srcpkgs/volumeicon/template
+++ b/srcpkgs/volumeicon/template
@@ -4,11 +4,16 @@ version=0.5.1
 revision=2
 build_style=gnu-configure
 configure_args="--enable-notify"
-hostmakedepends="pkg-config intltool"
-makedepends="alsa-lib-devel libnotify-devel gtk+3-devel"
+hostmakedepends="pkg-config intltool automake"
+makedepends="alsa-lib-devel libnotify-devel gtk+3-devel glib-devel"
 short_desc="Lightweight volume control that sits in your systray"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-3"
 homepage="http://softwarebakery.com/maato/volumeicon.html"
 distfiles="http://softwarebakery.com/maato/files/volumeicon/volumeicon-${version}.tar.gz"
 checksum=24b8c1d0a81d708b201ce6e67301fc175d65588d892d01859f667b8db8a05da0
+
+# necessary because of notify patch
+pre_configure() {
+    autoreconf -fi
+}
```

But it errors out with weird stuff

```
configure.ac:18: warning: AM_NLS is m4_require'd but not m4_defun'd
/usr/share/aclocal/intltool.m4:27: IT_PROG_INTLTOOL is expanded from...
/usr/share/aclocal/intltool.m4:209: AC_PROG_INTLTOOL is expanded from...
configure.ac:18: the top level
configure.ac:18: warning: AM_NLS is m4_require'd but not m4_defun'd
aclocal.m4:513: IT_PROG_INTLTOOL is expanded from...I can't find what
aclocal.m4:695: AC_PROG_INTLTOOL is expanded from...
configure.ac:18: the top level
configure.ac:18: warning: AM_NLS is m4_require'd but not m4_defun'd
aclocal.m4:513: IT_PROG_INTLTOOL is expanded from...
aclocal.m4:695: AC_PROG_INTLTOOL is expanded from...
configure.ac:18: the top level
configure:4774: error: possibly undefined macro: AM_NLS
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
autoreconf: /usr/bin/autoconf failed with exit status: 1
```

**Comment by ericonr** [comment]
@sarether do you know how to use `xbps-src`? If you could test my PR above, that'd be great c:

**Comment by sarether** [comment]
> @sarether do you know how to use `xbps-src`? If you could test my PR above, that'd be great c:

I installed Void and used it for about a week. Now I got only the basic use of XBPS. I've used Arch Linux, Slackware, and some other distributions for years.
I'll test it in the coming days.
Thanks.

**Comment by sgn** [comment]
Fixed by #23367

**Comment by sarether** [comment]
Thanks for fix update.
